src/main/java/net/sacredlabyrinth/phaed/simpleclans/Helper.java

```diff
@@ -662,7 +663,7 @@ public static String formatClanChat(ClanPlayer cp, String msg, Map<String, Strin
                 .replaceAll("%nick-color%", (cp.isLeader() ? leaderColor : memberColor))
                 .replaceAll("%player%", cp.getName())
                 .replaceAll("%rank%", rankFormat)
-                .replaceAll("%message%", msg);
+                .replaceAll("%message%", Matcher.quoteReplacement(msg));
```

Comment:
since it seems using regex isn't desired, the method that should be used here is `replace()` not `replaceAll().` This will also negate the need to do `Matcher.quoteReplacement()`

Reply:
You're right

Reply:
Additionally, since avoiding replaceAll avoids a costly regex computation, you'll save on cpu cycles doing just a `replace` as well.